src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/WhereaboutsConfig.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.justice.digital.hmpps.whereabouts.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Whereabouts Details
+ **/
+@SuppressWarnings("unused")
+@ApiModel(description = "Whereabouts Details")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+public class WhereaboutsConfig {
+
+    @NotNull
+    private boolean enabled;
+
+    /**
+     * Whether this prison is enabled for whereabouts
+     */
+    @ApiModelProperty(required = true, value = "Whether this prison is enabled for whereabouts")
+    @JsonProperty("enabled")
+    public boolean getEnabled() {
+        return enabled;
+    }
+
+    public void setEnabled(final boolean enabled) {
+        this.enabled = enabled;
+    }
+
+    @Override
+    public String toString() {
+        final var sb = new StringBuilder();
```

Comment:
Does this need to be a non-standard toString?

Reply:
I can't see why, just copied it blindly from elite2.  I'll change it.